docker/testing/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Docker file to test full-stack.t in a travis like environment
+FROM ubuntu:12.04
+run apt-get -y update
+run apt-get -y install python-software-properties
+run add-apt-repository -y ppa:kubuntu-ppa/backports
+run apt-get -y update
+run apt-get -y install libdbus-1-dev libssh2-1-dev libopencv-dev libtheora-dev libcv-dev libhighgui-dev tesseract-ocr libsndfile1-dev libfftw3-dev qemu-system automake libtool libxml2-dev libgmp3-dev ruby-sass git libssl-dev libgcrypt11-dev curl wget
+run curl http://stephan.kulow.org/phantomjs-2.1.1-linux-x86_64.tar.bz2 | tar xvjf -
```

Comment:
in case you wonder: I just mirrored it as the official download link is very unreliable